### PR TITLE
improvement: update with new persist methods

### PR DESCRIPTION
**Pull request by sentinel1909**
This is a start to updating the docs with the new methods for Persist, to go along with issue #169. There is more to do, ideally I'd like to create an example or two. I thought it might be best to get this basic info up fairly quickly and then go from there.

Side note: I changed the sub-heading under resources from Databases to Data Persistence, I thought it was more descriptive.

### CODE REVIEW

File: mint.json [hunk 1]
```diff
@@ -70,7 +68,7 @@
       "pages": [
         "resources/overview",
         {
-          "group": "Databases",
+          "group": "Data Persistence",
```
Comment (jonaro00) [inline mint.json:71]: WDYT? @ivancernja

File: resources/shuttle-persist.mdx [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
 
 ## Usage
-To use the persist plugin, add `shuttle-persist` to the dependencies of your service with `cargo add shuttle-persist`.  Then, annotate your `shuttle_runtime::main` function by adding the `shuttle-persist::Persist` attribute as a parameter.  This will give a `PersistInstance`. There are two methods available to `PersistInstance`,  `.save()` and `.load()`. Objects can be saved or loaded with these two methods. 
+
+To use the persist plugin, add `shuttle-persist` to the dependencies of your service with `cargo add shuttle-persist`. Then, annotate your `shuttle_runtime::main` function by adding the `shuttle-persist::Persist` attribute as a parameter. This will give a `PersistInstance`. There are six methods available to `PersistInstance`, providing tools to manage the data stored within. The methods are:
+
+- `clear()`: removes the keys within the `PersistInstance`
+- `list()`: returns a vector of strings containing all the keys associated with a `PersistInstance`
+- `load()`: loads the contents of the `PersistInstance`
```
Comment (jonaro00) [inline resources/shuttle-persist.mdx:13]: "loads a value from a key" or something like that

**Review by jonaro00 — COMMENTED** [review]
Minor comments

File: resources/shuttle-persist.mdx (hunk 2 above)
Comment (sentinel1909) [reply resources/shuttle-persist.mdx]: Done, made the revision just now and pushed an update.

File: mint.json (hunk 1 above)
Comment (sentinel1909) [reply mint.json:71]: Can revert this, if not desirable, it was just a thought.

**Comment by sentinel1909** [comment]
Anything further to do on this one?

File: mint.json (hunk 1 above)
Comment (ivancernja) [reply mint.json:71]: @jonaro00 @sentinel1909 I'm not a fan of it, to be honest. I feel like "Databases" would resonate better with both beginners and experienced developers whereas "Data Persistance" might put some beginners off. Plus, if we look at other platforms, iirc they all use the term "Databases".

Comment (sentinel1909) [reply mint.json]: Understood, fixed. Name changed back to "Databases"

**Pull request merged**